### ISSUE TITLE
double click on the error message to jump to the actual code in the editor

### ISSUE THREAD

**Issue by asmwarrior** [vectorgraphics/asymptote-server#1]
Hi, the site http://asymptote.ualberta.ca  works great!
I would suggest that if it can add one feature, that if there are some build errors, it could help the user to locate the error in the source code quickly, thanks!

**Comment by johncbowman** [comment]
Yes, this is a good idea and will be possible once we adopt CodeMirror. For now, I've implemented something more primitive in the terminal window that looks a bit like this:
real a=b;
_____ ^
workspace_1.asy: 1.8: no matching variable of name 'b'
